build.gradle

```diff
@@ -153,7 +153,6 @@ android {
         buildConfigField "String", "COMMIT_HASH", "\"${buildCommit}\""
         buildConfigField "boolean", "ENABLE_AUTO_INSTALLS_LAYOUT", "false"
 
-        resValue("string", "launcher_component", "${applicationId}/app.lawnchair.LawnchairLauncher")
```

Comment:
```xml
<string name="launcher_component" translatable="false">null/app.lawnchair.LawnchairLauncher</string>
```

Reply:
`launcher_component` generating should be deferred, with different `applicationId` in several product flavors.